README.md

```diff
@@ -22,41 +22,90 @@ func sayHelloToCat(kittenName: String) {
 
-*  At this stage, where the student has NOT seen a function yet. It's important to be explicit in our descriptions of the various elements that make up a function. Almost like operating in a slo-mo like manner, making NO assumptions.
-* Readme narrative is around puppies and kittens (include this gif? - http://i.imgur.com/85Pd59R.gif)
-* Put the student in the shoes of a dog. If they were a dog, there is certain functionality that a dog walks around with. They can bark, they can eat, etc. BUT, we're special.. we can speak English.
-* We can encapsulate within a function, a print statement that will greet others (our personalized greeting) and we don't want to have to type this over and over and over again where we can make a mistake or make typo's, instead we can create a function that within the scope of that function.. it will print out the greeting we want. Then we can call on that function over and over.
+After the `func` keyword, you type the _name_ of the function, just like a variable or constant. You'll use this name later on to _call_ the function (you'll see an example of this in action soon). The name can be anything you want—just like a variable! Normally, functions in Swift are named in _camel case_, which that it starts with a lowercase letter, and each word in the function's name is capitalized, like you see in `sayHello`. (Swift's function names cannot contain spaces.)
 
+After the name of the function, you type a set of parentheses: `()`. For now, that set of parentheses will be empty. You'll soon see why you type those parentheses, but for now, don't worry about them.
+
+After the parentheses comes a set of curly braces: `{}`. The first one, `{`, opens up a _code block_. This code block is also referred to as the function's _body_. Inside this code block is where you write what you want your function to do. You end the function with a matching curly brace: `}`. This lets Swift know that your function definition is done.
+
+Inside those curly braces is where the magic happens. You write exactly what you want your new function to do. It's pretty much what you've been doing in the past lessons, but now it's inside a function. In `sayHello()`, the only thing you do is declare a constant string called `greeting`, then print that string to the console by calling the `print()` function.
+
+## Calling Functions
+
+When you _call_ a function, your program jumps to that function that executes the bit of code inside the function's body (denoted by the curly braces). You usually call a function outside of the function definition itself. A function is called simply by writing the name of the function followed by parentheses. Try entering this in a new playground file:
 
 ```swift
 func sayHello() {
-    print("Hello everyone!")
+    let greeting = "Hello there!"
+    print(greeting)
 }
+
+sayHello()
+sayHello()
 ```
```

Comment:
I like the idea of showing the student what it is we expect to print to the console if we are to call on a print function like you did.

```swift
func sayHello() {
    print("Hello everyone!")
    let greeting = "Hello there!"
    print(greeting)
}

sayHello()
// prints "Hello everyone!"
// prints "Hello there!"

sayHello()
// prints "Hello everyone!"
// prints "Hello there!"
```